### PR TITLE
Initialize promise within a Worker

### PR DESCRIPTION
**Pull request by k33g**

### CODE REVIEW

File: src/main/golo/async.golo [hunk 1]
```diff
@@ -85,6 +85,17 @@ augment gololang.concurrent.async.Promise {
     }): start()
     return this: future()
   }
+
+  ----
+  You can define a promise which runs "stuff" inside a Worker
```
Comment (jponge) [inline src/main/golo/async.golo]: Sentences shall end with `.`

File: src/main/golo/async.golo [hunk 2]
```diff
@@ -85,6 +85,17 @@ augment gololang.concurrent.async.Promise {
...
+  You have to pass a `WorkerEnvironment` to `initializeWithinWorker` method
+  ----
+  function initializeWithinWorker = |this, env, closure| {
```
Comment (jponge) [inline src/main/golo/async.golo]: You should provide a list description of the arguments.

File: src/main/golo/async.golo [hunk 3]
```diff
@@ -85,6 +85,17 @@ augment gololang.concurrent.async.Promise {
...
+    env: spawn(|message| {
+      this: initialize(closure)
+    }): send("")
```
Comment (jponge) [inline src/main/golo/async.golo]: Why send an empty string?

Comment (k33g) [inline src/main/golo/async.golo]: What can I send?
null generate an error
and I can't find a use case where the message is useful

**Pull request closed without merging**